Improve WEB_DOMAIN documentation (#817)
tootsuite/documentation: content/en/admin/config.md

#### `WEB_DOMAIN`

It is possible to run the Mastodon interface on one domain, while having the users' handles on a different domain, e.g. addressing users as `@alice@example.com` but accessing Mastodon on `mastodon.example.com`. This may be useful if your domain name is already used for a different website but you still want to use it as a Mastodon identifier because it looks better/shorter.

> As for `LOCAL_DOMAIN`, it cannot be safely changed once set.

Reviewer: ```suggestion
As with `LOCAL_DOMAIN`, `WEB_DOMAIN` cannot be safely changed once set.
```
